src/Microsoft.Azure.ServiceBus/Management/NamespaceType.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    /// <summary>
+    /// Specifies the type of entities the namespace can contain.
+    /// </summary>
+    public enum NamespaceType
+    {
+        /// <summary>
+        /// Namespace contains messaging entities (queues, topics)
+        /// </summary>
+        Messaging = 0,
```

Comment:
Technically, both ASB and Notification Hubs are messaging.
As a user, I'd affiliate more with `ServiceBus` namespace, contrasted with `Mixed`. `Mixed` would indicate Service Bus and Notification Hubs.